app/controllers/application_controller.rb

```diff
@@ -1,2 +1,12 @@
 class ApplicationController < ActionController::Base
+    helper_method :current_user
+    helper_method :logged_in?
+
+    def current_user
+        User.find_by(id: session[:user_id])
```

Comment:
@idprietop details matter when you are writing code, please fix this code indentation and make sure you have your editor configured to automatically manage this.

Reply:
I will make sure that the editor is configured to manage the code automatically and I will take into account the details when writing the code.